File: Java e JUnit/Introducao ao JUnit/Testes/CreditoTest.java
import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

class CreditoTest {
	
	Credito credito = new Credito();

	@Test
	void testConcede() {
		assertTrue(credito.concedeCredito(12, true, 5001));
	}
	
	@Test
	void testNaoConcedePorParcelasMaiorQue12() {
		assertFalse(credito.concedeCredito(13, true, 5001));
	}
	
	@Test
	void testNaoConcedePorParcelasMenorQue12MaiorQue0() {
		assertFalse(credito.concedeCredito(0, true, 5001));
	}
	
	@Test
	void testNaoConcedePorParcelasMenorQue0() {
		assertFalse(credito.concedeCredito(-2, true, 5001));
	}
	
	@Test
	void testNaoConcedePorNaoConfiavel() {
		assertFalse(credito.concedeCredito(5, false, 5001));
	}
	
	@Test
	void testNaoConcedePorSalario() {
		assertFalse(credito.concedeCredito(12, true, 2500));
	}
	
	@Test
	void testDefinirCredito2000() {
		assertEquals(20000, credito.definirCredito(12, true, 5001));
	}
	
	@Test
	void testDefinirCredito0() {
		assertEquals(0, credito.definirCredito(13, true, 5000));
	}

}

File: Python e PyTest/Introducao ao PyTest/tests/Credito_test.py
from classes import Credito

def test_concede():
    assert Credito.concederCredito(12, True, 5001)

def test_NaoConcedePorParcelasMaiorQue12():
    assert Credito.concederCredito(13, True, 5001)

def test_NaoConcedePorParcelasIgualA0():
    assert Credito.concederCredito(0, True, 5001)

def test_NaoConcedePorParcelasMenorQue0():
    assert Credito.concederCredito(-2, True, 5001)

def test_NaoConcedePorNaoConfiavel():
    assert Credito.concederCredito(5, False, 5001)

def test_NaoConcedePorSalarioMenorQue5000():
    assert Credito.concederCredito(12, True, 2500)

def test_DefinirCredito2000():
    assert Credito.concederCredito(12, True, 5001)

def test_DefinirCredito0():
    assert Credito.concederCredito(13, True, 5001)
